CHANGELOG.md

```diff
@@ -34,6 +39,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - activation checkpoint: Added a context manager to disable checkpoint in case the same wrapped module
                          needs to be checkpointed and not checkpointed in different parts of
                          the module forward pass. [#772]
+- FSDP: Added a toggle with an environment variable ENABLE_NCCL_BASE_COLLECTIVES=[0,1] to allow users
```

Comment:
typo: disable.
Also mention which behavior is on by default.

Reply:
Done with a new commit